### DISCUSSION TITLE
Topical Timestamp

### DISCUSSION THREAD

**Discussion by 0x5am5** [deepgram/community#122] [General help] [answered]
I'm using the new Nova model and can see I'm indeed getting timestamps per-word. 

I wondered if there was a way to apply this to topics. Where the topic begins / ends would be fab.

**Comment by incubatize** [comment, marked as answer]
Hi @fakesamgregory  - 
Thanks for the great feedback! Our current topic detection feature provides start and end word indices out-of-the-box. And, since each word has an associated timestamp value, you should be able to get the timestamp for a specific section of topics. Is that what you are looking for?

**Reply by 0x5am5** [reply]
This is exactly what I'm looking for. Thank you! Ideally I'd like it a bit more simple, tied to the actual topic but this is by no means impossible.
